helm_deploy/manage-hmpps-auth-accounts/templates/ingress.yaml

```diff
@@ -12,6 +12,11 @@ metadata:
     heritage: {{ .Release.Service }}
   annotations:
     kubernetes.io/ingress.class: "nginx"
+    # note this limit is per ingress replica
+    nginx.ingress.kubernetes.io/limit-rpm: 10
```

Comment:
10 per minute?  Seems a bit tight.

Reply:
yes initially I thought that too - but remember it's per ingress instance.  Each request round-robins over many ingress instances.

Reply:
Also there is a default burst value of 10x that number.

Reply:
Fair enough. I imagine there will be some fine tuning required whatever you pick.